nc_time_axis/__init__.py

```diff
@@ -233,14 +248,20 @@ def convert(cls, value, unit, axis):
                 return value
             first_value = value
 
-        if not hasattr(first_value, 'calendar'):
-            raise ValueError('A "calendar" attribute must be attached to '
-                             'netcdftime object to understand them properly.')
-        return netcdftime.date2num(value, cls.standard_unit,
-                                   calendar=first_value.calendar)
+        if not isinstance(first_value.datetime, netcdftime.datetime):
```

Comment:
@lbdreyer I guess at this point you're expected the `first_value` to be a `CalendarDateTime`, otherwise the `first_value.datetime` might/will raise an `AttributeError` exception. Did you want to also consider that here? Is that appropriate?